Sources/StreamChatSwiftUI/ChatChannel/Composer/Suggestions/CommandsConfig.swift

```diff
@@ -33,23 +33,41 @@ public class DefaultCommandsConfig: CommandsConfig {
     public func makeCommandsHandler(
         with channelController: ChatChannelController
     ) -> CommandsHandler {
-        let mentionsCommand = MentionsCommandHandler(
+        let mentionsCommandHandler = MentionsCommandHandler(
             channelController: channelController,
             commandSymbol: mentionsSymbol,
             mentionAllAppUsers: false
         )
-        let giphyCommand = GiphyCommandHandler(commandSymbol: "/giphy")
-        let muteCommand = MuteCommandHandler(
-            channelController: channelController,
-            commandSymbol: "/mute"
-        )
-        let unmuteCommand = UnmuteCommandHandler(
-            channelController: channelController,
-            commandSymbol: "/unmute"
-        )
-        let instantCommands = InstantCommandsHandler(
-            commands: [giphyCommand, muteCommand, unmuteCommand]
+        
+        var instantCommands = [CommandHandler]()
+        
+        let channelConfig = channelController.channel?.config
+        
+        let giphyEnabled = channelConfig?.commands.first(where: { command in
+            command.name == "giphy"
+        }) != nil
```

Comment:
I think `.contains` would make more sense here instead of `.first`, right?

```suggestion
        let giphyEnabled = channelConfig?.commands.contains(where: { command in
            command.name == "giphy"
        })
```

Reply:
I've tried this, it still requires the != nil check, so I would rather keep it like this.